spec/features/batch_context/globus_spec.rb

```diff
@@ -8,7 +8,7 @@
   before do
     globus_dir.rmtree if globus_dir.directory?
     globus_dir.mkpath
-    allow(Settings.globus).to receive(:directory).and_return(globus_dir)
+    allow(Settings.globus).to receive(:directory).and_return(globus_dir.to_s)
 
```

Comment:
The settings have a string, not a path.  We need a string for our new test to work ... so this makes the mock match reality.